Source/Android/arcana/threading/task_schedulers.h

```diff
@@ -66,23 +61,22 @@ namespace arcana
 
         looper_scheduler& operator=(looper_scheduler&& other)
         {
+            reset();
+
             m_looper = other.m_looper;
             m_fd[0] = other.m_fd[0];
             m_fd[1] = other.m_fd[1];
 
             other.m_looper = nullptr;
-            other.m_fd[0] = 0;
-            other.m_fd[1] = 0;
```

Comment:
Why don't we need these any more? What happens if `other` is invoked after `this` has taken over?

Reply:
Right, good point. I should keep this.

Reply:
Apparently -1 is an invalid file descriptor as that's what gets returned by `open` for example when it fails.